documentation/applying_manifest_blocks.md

```diff
@@ -403,97 +403,93 @@ Create a manifest that sets up a web server with nginx, and run it as a plan.
 - [Bolt Projects](projects.md)
 
 
-### Create a sample manifest for IIS on Windows
+### Create a Bolt plan for IIS on Windows
 
 Create a manifest that sets up a web server with IIS and run it as a plan.
 
-1.  Go to the `site-modules` directory in the default Bolt project directory:
-    `~/.puppetlabs/bolt/site-modules`
-
-1.  Create a module named profiles.
-    -   If you use the Puppet Development Kit: `pdk new module profiles`
-    -   Otherwise create `~/.puppetlabs/bolt/site-modules/profiles`
-
-1.  Add a `plans` directory to the profiles module.
-
-1.  Install the IIS dependencies.
-    1.  Add the following code to `~/.puppetlabs/bolt/Puppetfile`
-
-        ```ruby
-        forge 'http://forge.puppetlabs.com'
-        mod 'puppetlabs-iis', '4.3.2'
-        mod 'profiles', local: true
-        ```
-
-    1.  Run `bolt puppetfile install`
-
-1.  In the plans directory, create a manifest file called `iis_install.pp` and
-    add the following code:
-
-    ```puppet
-    plan profiles::iis_install(
-         TargetSpec $targets,
-         String $site_content = 'hello!',
-       ) {
-
-         # Install the puppet-agent package if Puppet is not detected. 
-         # Copy over custom facts from the Bolt modulepath.
-         # Run the `facter` command line tool to gather target information.
-         $targets.apply_prep
-
-         # Compile the manifest block into a catalog
-         return apply($targets, '_catch_errors' => true) {
-           $iis_features = ['Web-WebServer','Web-Scripting-Tools']
-
-           iis_feature { $iis_features:
-             ensure => 'present',
-           }
-
-           # Delete the default website to prevent a port binding conflict.
-           iis_site {'Default Web Site':
-             ensure  => absent,
-             require => Iis_feature['Web-WebServer'],
-           }
+1. Create a new Bolt project named 'profiles' by running `New-BoltProject -Name profiles` if using
+   PowerShell or `bolt project init profiles` inside the directory you want to make a Bolt project.
```

Comment:
```suggestion
1. Create a new Bolt project named 'profiles' by running `bolt project init profiles` or
  `New-BoltProject -Name profiles` if using PowerShell inside the directory you want to make a Bolt project.
```

Reply:
This is literally how I had it and Heston asked me to change it https://github.com/puppetlabs/bolt/pull/2590#discussion_r566244010 going to go with his suggestion!

Reply:
hehe. The reason I switched the phrases around was because I thought the "if" conditional followed by "inside..." felt awkward to read. But now that I'm reading this with fresh eyes, and because it clearly still feels a bit awkward, I think I'm going to suggest a different approach. (Sidenote: This is why I selfishly liked the old command that created a Bolt directory for you 😆 ) 

```suggestion
1. Create a directory to use as your Bolt project or use an existing directory. 
1. Convert the directory into a Bolt project named 'profiles' by running `bolt project init profiles`, or `New-BoltProject -Name profiles` if you're using PowerShell.
```